task-management/src/main/resources/locale/portlet/taskManagement_en.properties

```diff
@@ -160,7 +160,7 @@ popup.msg.projectNotExist=This project does not exist or has been deleted.
 popup.msg.deleteProject=You are about to delete project <strong>{}</strong> and all its tasks. Please confirm.
 popup.msg.deleteLabel=Are you sure you want to delete <strong>{0}</strong> label?
 popup.msg.cloneProject=You are about to clone project <strong>{}</strong>. Please confirm.
-popup.msg.deleteTask=Are you sure you want to delete this task : '{}' ?
```

Comment:
we must correct that in Crowdin

Reply:
It will be generated automatically in crowdin, but we have to fix the key value in English language